src/main/resources/db/migration/V1_4__alter_table_aktivitetskrav_vurdering.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE AKTIVITETSKRAV_VURDERING
+DROP COLUMN tilfelle_start;
+
+ALTER TABLE AKTIVITETSKRAV_VURDERING
+ADD COLUMN updated_by VARCHAR(7);
+
+ALTER TABLE AKTIVITETSKRAV_VURDERING
+ADD COLUMN stoppunkt_at DATE;
```

Comment:
Er litt usikker på hva som er best å bruke for datoer i psql? Men ser at vi har brukt `timestamp` eller `timestampz` rundt omkring i noen andre apper 🤔

Reply:
Her trenger vi bare dato, og ikke tidsinformasjon, så da tror jeg det er vanlig med DATE. Ser i hvert fall sånn ut i andre apper 🤷‍♂️

Reply:
Ellers bruker vi nå konsekvent timestampz for når vi legger til nye tidsstempel-kolonner (og OffsetDateTime i Kotlin), men i noen app'er (feks isdialogmote) forekommer timestamp (og LocalDateTime).